mne/preprocessing/interpolate.py

```diff
@@ -98,48 +105,86 @@ def interpolate_bridged_electrodes(inst, bridged_idx):
     --------
     mne.preprocessing.compute_bridged_electrodes
     """
+    from scipy.sparse.csgraph import connected_components
+
     _validate_type(inst, (BaseRaw, BaseEpochs, Evoked))
+    _validate_type(bad_limit, "int", "bad_limit")
```

Comment:
There are many int-like things, this will get you an `int` back which is nice
```suggestion
    bad_limit = _ensure_int(bad_limit, "bad_limit")
```